unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEditorHUDController.cs

```diff
@@ -1,43 +1,210 @@
+using DCLServices.WearablesCatalogService;
 using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
 
 namespace DCL.Backpack
 {
     public class BackpackEditorHUDController : IHUD
     {
+        private UserProfile ownUserProfile => userProfileBridge.GetOwn();
+
         private readonly IBackpackEditorHUDView view;
         private readonly DataStore dataStore;
+        private readonly IUserProfileBridge userProfileBridge;
+        private readonly IWearablesCatalogService wearablesCatalogService;
+        private readonly IBackpackEmotesSectionController backpackEmotesSectionController;
+        private readonly BackpackEditorHUDModel model = new ();
+        private bool avatarIsDirty;
+        private float prevRenderScale = 1.0f;
 
-        public BackpackEditorHUDController(IBackpackEditorHUDView view, DataStore dataStore)
+        public BackpackEditorHUDController(
+            IBackpackEditorHUDView view,
+            DataStore dataStore,
+            IUserProfileBridge userProfileBridge,
+            IWearablesCatalogService wearablesCatalogService,
+            IBackpackEmotesSectionController backpackEmotesSectionController)
```

Comment:
i dont think we need a layer of abstraction/interface for an internal controller, you could just inject the concrete class